src/App.css

```diff
@@ -36,3 +36,27 @@
     transform: rotate(360deg);
   }
 }
+
+ul{
```

Comment:
I dont think we need to keep these styles here, since they are not decided to be global. We can use them where needed and refactor or add them here once we decide that

Reply:
these are general css , which we will be needing everywhere inside app. So as we are importing app.css in our routes file so every children component as access to these css without importing and reducing code repeatability. We will discuss this in todays call and implement accordingly